lib/runtime/life/exports_test.go

```diff
@@ -193,6 +193,7 @@ func TestInstance_FinalizeBlock_NodeRuntime(t *testing.T) {
 }
 
 func TestInstance_ExecuteBlock_GossamerRuntime(t *testing.T) {
+	t.Skip() // todo determine why this test is failing
```

Comment:
what is the current failure message? if it passed before and isn't now, maybe there was some update in the `ext_`s that were previously here that now causes something incorrect to happen

Reply:
It's failing with: 
```
DBUG[08-05|13:09:36] [ext_storage_root_version_1]             pkg=runtime module=perlin/life root=0x04dd9640566655b942d96f951f3f2d661eb76d7d8286b8443baf9a07f6d8d4d0 caller=resolver.go:537
DBUG[08-05|13:09:36] [ext_storage_changes_root_version_1] returning None pkg=runtime module=perlin/life caller=resolver.go:550
--- FAIL: TestInstance_ExecuteBlock_GossamerRuntime (0.94s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0xa25f64]

goroutine 6 [running]:
testing.tRunner.func1.2(0xb89ee0, 0x108d090)
	/usr/local/go/src/testing/testing.go:1143 +0x332
testing.tRunner.func1(0xc000001b00)
	/usr/local/go/src/testing/testing.go:1146 +0x4b6
panic(0xb89ee0, 0x108d090)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
github.com/ChainSafe/gossamer/lib/runtime/life.(*Instance).ExecuteBlock(0xc0039685e0, 0xc003da8220, 0xc000c968d0, 0x0, 0x0, 0x0, 0x0)
	/home/emack/projects/ChainSafe/gossamer/lib/runtime/life/exports.go:134 +0x84
github.com/ChainSafe/gossamer/lib/runtime/life.TestInstance_ExecuteBlock_GossamerRuntime(0xc000001b00)
	/home/emack/projects/ChainSafe/gossamer/lib/runtime/life/exports_test.go:209 +0x1d0
testing.tRunner(0xc000001b00, 0xca5008)
	/usr/local/go/src/testing/testing.go:1193 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1238 +0x2b3


Process finished with the exit code 1
```

Yes, there may be a ext_ that was updated that breaks this, it seems to break at ext_storage_changes_root_version_1, so maybe there is an issue with that (I don't know whan storage_changes_root is supposed to do).

Reply:
Update: these work now, after updating instance to error instead of panic on function not found, so then inst.version code is now un-commented, which fixes these.